pages/calendar/components/Calendar/CalendarHeader/index.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from "classnames/bind";
+import dayjs from "dayjs";
+
+import MonthNavButton from "@/components/DatePicker/MonthNavButton";
+import { CalendarType } from "@/pages/calendar/index.page";
+import { useDateStore } from "@/store/useDateStore";
+
+import styles from "./CalendarHeader.module.scss";
+import CalendarTypeSegmentedButton from "../../CalendarTypeSegmentedButton";
+
+const cn = classNames.bind(styles);
+
+interface CalendarHeader {
+  setCalendarType: (type: CalendarType) => void;
+}
+
+export default function CalendarHeader({ setCalendarType }: CalendarHeader) {
+  const { viewDate, setViewDate, setFocusDate } = useDateStore();
+
+  const handleTodayClick = () => {
+    const today = dayjs();
+    setViewDate(today);
+    setFocusDate(today);
```

Comment:
```suggestion
    setViewDate(dayjs());
    setFocusDate(dayjs());
```
아까 위에서는 따로 변수 안만들고 했던 것 같아요! 직접 넣어도 괜찮을 것 같네요 ㅇ_ㅇ